server/auth/login.py

```diff
@@ -62,9 +73,9 @@ def login_request(self, packet, address):
         permission_error = 'You do not have permission to log in to this server' if auth_status == 'not_permitted' else ''
 
         if uid:
-            self.server.handle_until_value('auth:set_address', uid, address)
-            new_subscriber = self.server.handle_until_value('auth:new_subscriber', True, uid)
-            ftp = self.server.handle_until_value('auth:free_to_play', True, uid)
+            self.server.handle_until_value('auth:set_address', True, uid, address)
```

Comment:
This can use the regular `handle` function, as I don't see a reason to prevent execution of later modules at any point